iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/TwoWindingsTransformerFortescue.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.powsybl.commons.extensions.Extension;
+import com.powsybl.iidm.network.TwoWindingsTransformer;
+
+/**
+ * @author Jean-Baptiste Heyberger <jbheyberger at gmail.com>
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public interface TwoWindingsTransformerFortescue extends Extension<TwoWindingsTransformer> {
+
+    String NAME = "twoWindingsTransformerFortescue";
+
+    @Override
+    default String getName() {
+        return NAME;
+    }
+
+    double getRo();
+
+    void setRo(double ro);
+
+    double getXo();
+
+    void setXo(double xo);
+
+    boolean isFreeFluxes();
```

Comment:
Free fluxes must be documented.

Reply:
a java doc will follow when homopolar LfModel will be transferred from incubator.
But user doc should also follow